external/audiocodec/aacdec/fxp_mul32_c_equivalent.h

```diff
@@ -77,7 +77,7 @@ extern "C"
     }
 
 
-    __inline  Int32 fxp_mul_16_by_16bb(Int32 L_var1,  Int32 L_var2)
+    static __inline  Int32 fxp_mul_16_by_16bb(Int32 L_var1,  Int32 L_var2)
```

Comment:
This is the external open source. Is this correct solution? Could you refer and check original source?